tests/tfm/tfm_regression_test/testcase.yaml

```diff
@@ -1,10 +1,11 @@
 common:
   tags: tfm
   build_only: true
-  platform_allow: nrf5340dk/nrf5340/cpuapp/ns nrf9160dk/nrf9160/ns
+  platform_allow: nrf5340dk/nrf5340/cpuapp/ns nrf9160dk/nrf9160/ns nrf54l15pdk/nrf54l15/cpuapp/ns
   integration_platforms:
     - nrf5340dk/nrf5340/cpuapp/ns
     - nrf9160dk/nrf9160/ns
+    - nrf54l15pdk/nrf54l15/cpuapp/ns
   harness: console
   harness_config:
     type: multi_line
```

Comment:
That's strange.

We didn't need to filter out

tfm.regression_ipc_lvl2.cc3xx:

?

How does this work?

Reply:
Because I haven't done yet  the validation of what is run on CI and what not...